rotkehlchen/db/snapshots.py

```diff
@@ -316,3 +289,9 @@ def delete(self, timestamp: Timestamp) -> Tuple[bool, str]:
             return False, 'No snapshot found for the specified timestamp'
         self.db.update_last_write()
         return True, ''
+
+    def add_nft_asset_ids(self, entries: List[str]) -> None:
+        """Add NFT identifiers to the DB to prevent unknown asset error."""
+        for entry in entries:
+            if entry.startswith(NFT_DIRECTIVE):
```

Comment:
Since add_asset_identifiers accepts multiple identifiers I would suggest to gather all entries that start with `NFT_DIRECTIVE` here and then pass them only once to self.db.add_asset_identifiers